Quran/SQLiteActiveTranslationsPersistence.swift

```diff
@@ -13,7 +13,7 @@ struct SQLiteActiveTranslationsPersistence: ActiveTranslationsPersistence, SQLit
 
     let version: UInt = 1
     var filePath: String {
-        return FileManager.default.documentsPath + "/translations.db"
+        return Files.databasesPath + "/translations.db"
```

Comment:
why not just have a single database with a column for downloaded, may make your life easier?